### ISSUE TITLE
Update ISCA/Trinity Tracking

### ISSUE THREAD

**Issue by Cotrina9** [gmo/grant#47]
Per Sarah - Google has updated their conversion policy for the grants.  I have to remove the conversion tracking for page load and add conversion tracking to the indicated decision button. I have the code, but I'm not sure how you add it to the page.  We will have to send Trinity the same thing too (since we don't have their credentials).

**Comment by celant12** [comment]
We got ISCA set for testing with the new code and we will package the code for trinity if the code works

**Comment by Cotrina9** [comment]
Per Sarah - I resubmitted ISCA's account through approval and it got kicked back due to the tracking code not being verified.

**Comment by mikegillis677** [comment]
Via sarah:
```
This is the message I got from Google:


Hi,

Thank you for reaching out to Ad Grants Support and for caring for account 181-294-4401. To reactivate your account, please do the following: 

    Verify the page on your website which has conversion tracking installed. Our system doesn’t show valid conversions because conversions may be limited to only views of your landing pages. To be compliant with this policy, conversions must not recorded on frequently viewed content pages, but rather be a more meaningful goal of your site.


Please reply when you’re ready for us to re-review your account.

Thank you,```

**Comment by Cotrina9** [comment]
We are still waiting on access from Trinity so this is moving to Sprint 16.

**Comment by Cotrina9** [comment]
We are still waiting on Trinity

**Comment by Cotrina9** [comment]
Per Jackie, Everything is running without new tracking.  WE'll address that before the year is over with all accounts that will need new conversion tracking on the page and Jackie will open a separate ticket to accommodate placing code on any account that needs it.
